src/pages/Profile/Photos/Photos.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import styles from "./Photos.module.css";
+
+const Photos = ({ photos }) => {
+  if (photos && photos.length > 0) {
+    return (
+      <div className="h-16 border-t border-gray-primary mt-12 pt-4">
+        <div className="grid grid-cols-3 gap-2">
+          {photos.map((photo) => (
+            <div key={photo.id} className="relative mr-6 mb-5">
+              <img
+                src={photo.src}
+                alt={photo.alt}
+                className="w-full h-full object-cover"
+                style={photo.style}
+              />
+              <div className="absolute bottom-0 left-0 bg-gray-200 z-10 w-full justify-evenly items-center h-full bg-black-faded group-hover:flex  ">
+                <div className="flex justify-center items-center ">
+                  <div className={styles["icons"]}>
+                    <p className="flex items-center text-white font-bold ">
+                      <svg
+                        xmlns="http://www.w3.org/2000/svg"
+                        viewBox="0 0 20 20"
+                        fill="currentColor"
+                        className="w-8 mr-2"
+                      >
+                        <path
+                          fillRule="evenodd"
+                          d="M3.172 5.172a4 4 0 015.656 0L10 6.343l1.172-1.171a4 4 0 115.656 5.656L10 17.657l-6.828-6.829a4 4 0 010-5.656z"
+                          clipRule="evenodd"
+                        />
+                      </svg>
+                      {Math.floor(Math.random() * 20)}
```

Comment:
Each post (photo) is supposed to have its own number of likes and comments. This is how it will be even coming from the backend (if exists).

This isn't a good way of displaying the numbers, because whenever you click on "Follow" the re-render will cause the random number to be re-evaluated, and thus, change, which I don't personally like.